Corgibytes.Freshli.Cli/Functionality/Cache.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using ServiceStack;
+
+namespace Corgibytes.Freshli.Cli.Functionality
+{
+    [Index(nameof(Key), IsUnique = true)]
+    public class CachedProperty
+    {
+        public int Id { get; set; }
+        public string Key { get; set; }
+        public string Value { get; set; }
+    }
+
+    class CacheContext : DbContext
+    {
+        public static readonly string cacheDirEnvVariable = "FRESHLI_CACHE_DIR";
```

Comment:
I like having the env variable as an option here. We should add a test for it, though. And we should clearly define whether or not the env variable overrides the command line parameter or vice-versa. I'm not sure what the convention should be there.

Reply:
I've added a comment to clarify: this is principally used internally to store and retrieve the user-provided `--cache-dir` argument from the database. However, I'll adjust the code so if the argument is NOT provided, and the environment variable is, it will use the environment variable's value.